python/paddle/fluid/incubate/fleet/collective/__init__.py

```diff
@@ -125,7 +150,182 @@ def save_persistables(self, executor, dirname, main_program=None):
             "In fleet.save_inference_model() function, main_program " \
             "must be as Program type."
 
-        io.save_persistables(executor, dirname, main_program, None)
+        io.save_persistables(executor, dirname, main_program, filename=filename)
+
+    def _save_train_status(self, path, train_status):
+        d = {}
+        d["epoch_no"] = train_status._epoch_no
+
+        file_name = "{}/fleet_train_status".format(path)
+        with open(file_name, 'w') as f:
+            json.dump(d, f)
```

Comment:
OK, that's what i mentioned before about persistable training status

Reply:
Same as above.